### PR TITLE
i18n(ko-KR): add `preact.mdx`

### PR DESCRIPTION
**Pull request by jsparkdev**
#### Description (required)

- add `preact.mdx`

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

File: src/content/docs/ko/guides/integrations-guide/preact.mdx [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+      include: ['**/solid/*'],
+    }),
+  ],
+});
+```
+
+## 예시
+
+* The [Astro Preact example](https://github.com/withastro/astro/tree/latest/examples/framework-preact) shows how to use an interactive Preact component in an Astro project.
+* The [Astro Nanostores example](https://github.com/withastro/astro/tree/latest/examples/with-nanostores) shows how to share state between different components — and even different frameworks! — in an Astro project.
```
Comment (dreyfus92) [inline src/content/docs/ko/guides/integrations-guide/preact.mdx:191]: This should be removed.

**Review by dreyfus92 — COMMENTED** [review]
Amazing job @jsparkdev, I just left a small suggestion 😁

**Comment by jsparkdev** [comment]
thank you! Problem solved.

**Review by dreyfus92 — APPROVED** [review]
Thank you @jsparkdev, LGTM ✅

**Pull request merged**